api/envoy/config/route/v3/route_components.proto

```diff
@@ -358,6 +358,14 @@ message Route {
   //    every application endpoint. This is both not easily maintainable and
   //    statistics use a non-trivial amount of memory(approximately 1KiB per route).
   string stat_prefix = 19;
+
+  // Sets the maximum size of a request body that will be buffered in memory. Envoy will return
+  // ``HTTP 413`` when buffer reaches the number set in this field.
```

Comment:
It isn't all the cases return 413
https://github.com/envoyproxy/envoy/blob/bd5a73e09a5f1ef993abf2415c6526172e3c97b5/source/common/http/filter_manager.cc#L544-L550

Reply:
@soulxu 
Thanks for pointing it out! Updated.